src/components/GenreList.vue

```diff
@@ -1,24 +1,32 @@
 <template>
   <div class="genre-list">
-    <GenreCard v-for="genre in genres" :key="genre.id" :genre="genre" />
+    <GenreCard
+      v-for="genre in genres"
+      :key="genre.id"
+      :genre="genre"
+    />
   </div>
 </template>
 
 <script>
+import { mapState, mapActions } from 'vuex';
 import GenreCard from "@/components/GenreCard.vue";
-import store from "@/store/index";
 
 // Bud -- I was trying to use mapState here, but it wasn't working...
 // Any ideas? Computed seems like the wrong way to do this
 export default {
-  computed: {
-    genres() {
-      return store.state.genre.genres;
```

Comment:
this way is a bit like doing rootState in redux.

the pattern is to use mapState and mapActions